validation/connector_plugin_manifest_latest.xsd

```diff
@@ -94,9 +94,40 @@
       <xs:pattern value="(\d+)((\.)(\d+))*((-)(\d+))*"/>
     </xs:restriction>
   </xs:simpleType>
+  <xs:group name="VendorInformation-G">
```

Comment:
For this VendorInformation-G group, we have a separate xsd file in monolith, https://opengrok/source/xref/teams_near/modules/XSD/VendorInfo.xsd. we probably want to keep it consistent with the monolith code.

Reply:
The update is just the copy from an updated generated file in near\tableau-1.3\res\FileFormat\connector_plugin_manifest_latest.xsd without the comments.

Reply:
Since I branched off of packaging, merging to dev will merge everything from packaging that has not yet been merged to dev.